release-notes/2.1/releases.json

```diff
@@ -1,13 +1,486 @@
 {
   "channel-version": "2.1",
-  "latest-release": "2.1.23",
-  "latest-release-date": "2020-10-13",
-  "latest-runtime": "2.1.23",
-  "latest-sdk": "2.1.811",
+  "latest-release": "2.1.24",
+  "latest-release-date": "2021-01-12",
+  "latest-runtime": "2.1.24",
+  "latest-sdk": "2.1.812",
   "support-phase": "lts",
   "eol-date": "2021-08-21",
   "lifecycle-policy": "https://dotnet.microsoft.com/platform/support/policy/",
   "releases": [
+    {
+      "release-date": "2021-01-12",
+      "release-version": "2.1.24",
+      "security": false,
```

Comment:
Not noted as a security release here but it is in `releases.json`.

Reply:
I updated releases-index,json. 2.1 is not a security release